jenkins/data-prepper/release-data-prepper-all-artifacts.jenkinsfile

```diff
@@ -83,11 +82,17 @@ pipeline {
                         destinationImage: "data-prepper:${VERSION}",
                         destinationRegistry: 'opensearchproject',
                     )
+                }
+            }
+        }
+        stage('Copy Docker Image to ECR') {
+            steps {
+                script {
                     copyContainer(
                         sourceImage: "data-prepper:${VERSION}-${DATA_PREPPER_BUILD_NUMBER}",
                         sourceRegistry: "${DATA_PREPPER_STAGING_CONTAINER_REPOSITORY}",
-                        destinationImage: "data-prepper:latest",
-                        destinationRegistry: 'opensearchproject',
+                        destinationImage: "data-prepper:${VERSION}",
```

Comment:
I'm thinking that we should still copy to the `latest` (for both DockerHub and ECR). Perhaps we should have a new job property which variably configures whether to do this or not. But, I'm also fine with having the manual step to copy to `latest` after this build runs.